includes/class-img-converter.php

```
@@ -462,6 +470,133 @@ private function convert_palette_to_truecolor( $image ) {
+						$img_info['lqip'] = array();
+					}
+
+					$img_info['dominant_color'][ $rel_path ] = $dominant_color;
+					if ( ! empty( $lqip ) ) {
+						$img_info['lqip'][ $rel_path ] = $lqip;
+					}
+
+					return $img_info;
+				}
+			);
+		}
```

PHPMD (2.15.0)

[error] The method store_placeholder_data is not named in camelCase. (undefined)

(CamelCaseMethodName)